packages/io-sign/src/document.ts

```diff
@@ -68,6 +69,11 @@ export const SignatureField = t.type({
 
 export type SignatureField = t.TypeOf<typeof SignatureField>;
 
+export enum FormFieldTypeEnum {
+  "SIGNATURE" = "SIGNATURE",
+  "OTHER" = "OTHER",
+}
```

Comment:
1. use a `union type` instead of an `enum`
2. avoid using "suffix" such as `Enum`, put directly the name of the type (`FormFieldType`)

```suggestion
export type FormFieldType = "SIGNATURE" | "OTHER"; 
```

moreover, if the only value allowed in our case is "SIGNATURE" make sure that a different value is never accepted at the outset (traces "OTHER" only if we do something with it later)